scripts/gen-certs.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0
+set -e
```

Comment:
We want to fail hard if there are unset variables.

Besides, it's good practice to pair `-e` with `set -o pipefail` to catch errors in command pipelines.  There are none here that I can see at the moment, but it costs little and would make the script more robust if `|`s are added in the future.

```suggestion
set -eu
set -o pipefail
```

Reply:
Thanks for the suggestions. Included.